src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -217,8 +218,31 @@ public Notifications notifications() {
     }
 
     @Override
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     public Iterable<Language> languages() {
-        return Collections.emptyList();
+        final RtJson rtJson = new RtJson(
+            this.request.uri()
+                .path("/languages")
+                .back()
+        );
+        final JsonObject json;
+        try {
+            json = rtJson.fetch();
+        } catch (final IOException ex) {
```

Comment:
I think it's better to let this exception bubble up and declare it in `throws` of the method. this is how all other methods in the library work